packages/persistencesniper/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=persistencesniper
+pkgver=v1.9.3.r0.g0bec863
+pkgrel=1
+groups=('blackarch' 'blackarch-defensive' 'blackarch-windows')
+pkgdesc='Hunt persistences implanted in Windows machines.'
+url='https://github.com/last-byte/PersistenceSniper'
+arch=('any')
+license=('MIT')
+makedepends=('git')
+source=("$pkgname::git+https://github.com/last-byte/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+package() {
+  cd $pkgname
+
+  install -dm 755 "$pkgdir/usr/share/windows/$pkgname"
```

Comment:
You should copy all MD files to the doc folder, copy the license file in the license folder and only copy `PersistenceSniper/` folder content in `/usr/share`

Reply:
Fixed.